### DISCUSSION TITLE
can i delete the old version

### DISCUSSION THREAD

**Discussion by sweiilu** [pear-devs/pear-desktop#2508] [Q&A] [answered]
![Screenshot 2024-10-15 153313](https://github.com/user-attachments/assets/50b30cb3-82d7-4480-9221-e96a8c53cefc)
![Screenshot 2024-10-15 153455](https://github.com/user-attachments/assets/188dc05b-a1e3-46d9-a7df-5b28d73a4fec)

**Comment by Lockszmith-GH** [comment, marked as answer]
This isn't a youtube-music question, it's about how you install it.
I'm unsure, but the folder structure looks like the one `scoop` created. You can use scoop's `cleanup` action.
